release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -496,6 +496,14 @@ In the following tables, features are marked with the following statuses:
 [id="ocp-4-14-openshift-cli-bug-fixes"]
 ==== OpenShift CLI (oc)
 
+Previously, container image references that have both tag and digest were not correctly interpreted by oc-mirror and resulted in an error message like below:
```

Comment:
Just a minor suggestion to change this to say "...error message like the one below:"

Reply:
Ack.